verifiers/utils/dsse.go

```diff
@@ -43,3 +90,127 @@ func DecodeSignature(s string) ([]byte, error) {
 
 	return nil, fmt.Errorf("%w: %v", serrors.ErrorInvalidEncoding, errs)
 }
+
+type SignatureEncoding int
+
+const (
+	// The DER signature is encoded using ASN.1
+	// (https://tools.ietf.org/html/rfc5480#appendix-A):
+	// ECDSA-Sig-Value :: = SEQUENCE { r INTEGER, s INTEGER }. In particular, the
+	// encoding is:
+	// 0x30 || totalLength || 0x02 || r's length || r || 0x02 || s's length || s.
+	SignatureEncodingDER SignatureEncoding = iota
+	// The IEEE_P1363 signature's format is r || s, where r and s are zero-padded
+	// and have the same size in bytes as the order of the curve. For example, for
+	// NIST P-256 curve, r and s are zero-padded to 32 bytes.
+	SignatureEncodingIEEE_P1363
```

Comment:
nit: This is maybe more [Go-like](https://go.dev/doc/effective_go#mixed-caps) but you could also add a [golangci-lint false positive comment](https://golangci-lint.run/usage/false-positives/).
```suggestion
	SignatureEncodingIEEEP1363
```